### ISSUE TITLE
Readme disclaimer of nextjs static only bundle 

### ISSUE THREAD

**Issue by davidcheung** [commit-app-playground/frontend-template#3]
We're using nextjs but not a full featured version, should document or link out to an explanation of what the differences are 
https://nextjs.org/docs/advanced-features/static-html-export#supported-features

basically we dont support server side rendering / api routes, and **only support static pages**

can also investigate if the bundle can be separated easily from nextjs to actually deploy the service to k8s + static pages on cdn

**Comment by shashtag** [comment]
Can you please elaborate more on the issue and what is required.

**Comment by davidcheung** [comment]
Yeah goal for now is just to update https://github.com/commit-app-playground/frontend-template/blob/main/templates/README.md so it lets users know even though Next.js supports server-side rendering and API routes, our template is designed only to build a static bundle therefore only supports https://nextjs.org/docs/advanced-features/static-html-export#supported-features
